driver/fixtures/fixtures.go

```diff
@@ -345,7 +348,7 @@ func (s *Suite) benchmarkTransform(b *testing.B, legacy bool) {
 	require.NoError(b, err)
 	defer dr.Close()
 
-	ctx, cancel := context.WithTimeout(context.Background(), parseTimeout)
+	ctx, cancel := context.WithTimeout(ctx, parseTimeout)
```

Comment:
The context isn't used till here, this can more simply be:

    ctx, cancel := context.WithTimeout(context.Background(), parseTimeout)